### PR TITLE
networkd: Add bridge port capabilities

### PR DESCRIPTION
**Pull request by toanju**
This PR adds the configuration switches for multicast flooding, neighbor
suppression and learning of a bridge port.

### CODE REVIEW

**Comment by yuwata** [comment]
Mostly good.
- Please also update test/fuzz/fuzz-netdev-parser/directives.netdev.
- CentOS seems not support some of the new flag. So, maybe the tests need to be conditionalized, i.e. if the file exit, then check the value.

**Comment by toanju** [comment]
I think you mean the `test/fuzz/fuzz-network-parser/directives.network` file since this is just for the bridge ports. Will add the options there.

I will add the checks for the according file in `/sys/...` as soon as I figured which ones are needed.

File: test/test-network/systemd-networkd-tests.py [hunk 1]
```diff
@@ -1284,7 +1284,11 @@ def test_bridge_property(self):
         self.assertEqual(self.read_bridge_port_attr('bridge99', 'dummy98', 'hairpin_mode'), '1')
         self.assertEqual(self.read_bridge_port_attr('bridge99', 'dummy98', 'path_cost'), '400')
         self.assertEqual(self.read_bridge_port_attr('bridge99', 'dummy98', 'unicast_flood'), '1')
+        if (os.path.exists('/sys/devices/virtual/net/bridge99/lower_dummy98/brport/multicast_flood')):
+          self.assertEqual(self.read_bridge_port_attr('bridge99', 'dummy98', 'mcast_flood'), '0')
```
Comment (poettering) [inline test/test-network/systemd-networkd-tests.py]: we indent python code by 4ch, see code around it

Comment (toanju) [reply test/test-network/systemd-networkd-tests.py]: thx. I update this as soon as I got some results from the ci

**Comment by yuwata** [comment]
> I think you mean the `test/fuzz/fuzz-network-parser/directives.network` file since this is just for the bridge ports. Will add the options there.

Ah, yeah.

**Pull request merged**